.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+
+services:
+  - docker
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - docker build --build-arg UNISON_VERSION=${UNISON_VERSION} --build-arg OCAML_VERSION=${OCAML_VERSION} -t ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:${tag} .;
+
+script:
+  - docker run -d ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:${tag} /bin/unison -version
+
+after_success:
+  - if [ "${TRAVIS_BRANCH}" = "master" ]; then
+      docker login -u="${DOCKER_HUB_USER}" -p="${DOCKER_HUB_PASS}";
+      docker push ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:${tag};
+      if [ "${tag}" = "${LATEST_TAG}" ]; then
+        docker tag ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:${tag} ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:latest;
+        docker push ${DOCKER_REPO_USERNAME}/${DOCKER_REPO_SLUG}:latest;
+      fi
+    fi
+
+notifications:
+  email: false
+
+env:
+  matrix:
+    - UNISON_VERSION=2.48.3 OCAML_VERSION=4.02 tag=2.48
+    - UNISON_VERSION=2.48.3 OCAML_VERSION=4.01 tag=2.48-4.01
```

Comment:
i'm thinking to make it clear what the second version number is, we should include `ocaml` in the name. e.g. `2.48-4.01` => `2.48.3-ocaml4.01`